pkg/certs/cert-inspection/certgraphapi/types.go

```diff
@@ -157,6 +157,8 @@ type CertKeyMetadata struct {
 	SignatureAlgorithm string
 	PublicKeyAlgorithm string
 	PublicKeyBitSize   string
+	NotBefore          string `json:"-"`
```

Comment:
mhm, `CertKeyMetadata` is defined in the library that may have many consumers with different expectations. Wouldn't it be better to allow consumers to control which fields they want to serialise instead ?